new-frontiers-14/frontier-station-14
# Eggnog

## About the PR



Adds eggnog as a craftable drink.

## Why / Balance


Woe, yuletide upon ye.

## How to test


1. Check the recipe in the guidebook.
2. Make some eggnog.

## Media


Christmas in medbay again.
![image](https://github.com/user-attachments/assets/7e4fe609-9eed-4e9e-b640-92a4be2c32f4)

Guidebook bar entry.
![image](https://github.com/user-attachments/assets/26c0f37e-19a1-43cc-a405-bdb7fc092891)

## Requirements



## Breaking changes


**Changelog**


:cl:
- add: Eggnog is now craftable by chefs and bartenders.
